sync/src/chain.rs

```diff
@@ -255,6 +257,7 @@ impl ChainSync {
 			blocks_total: match self.highest_block { Some(x) if x > self.starting_block => x - self.starting_block, _ => 0 },
 			num_peers: self.peers.len(),
 			num_active_peers: self.peers.values().filter(|p| p.asking != PeerAsking::Nothing).count(),
+			transaction_queue_pending: self.transaction_queue.lock().unwrap().status().pending,
```

Comment:
1. I can try that, but my gut feeling is that we will have to lock every field anyhow.

2. It's not possible to replace `Mutex` with `RwLock` because `SignedTransaction` is not `Sync` (because it has `Cell` internally). So I would have to copy all fields from `SignedTransation` to `VerifiedTransaction` and then back (when returning) or protect `SignedTransation` with `Mutex`.

Will try to tinker with it a little bit later.